app/middlewares/errors.js

```diff
@@ -6,7 +6,8 @@ const DEFAULT_STATUS_CODE = 500;
 const statusCodes = {
   [errors.DATABASE_ERROR]: 503,
   [errors.DEFAULT_ERROR]: 500,
-  [errors.INVALID_PASSWORD_ERROR]: 400
+  [errors.INVALID_PASSWORD_ERROR]: 400,
+  [errors.INVALID_EMAIL_ERROR]: 400
```

Comment:
Do you forget the USER_SIGNUP_ERROR,  API_JOKES_ERROR, ENCRYPT_PASSWORD_ERROR, EMAIL_EXISTING_ERROR right? Remember to select the status code correctly.